tools/grammar/src/test/resources/cypher.txt

```diff
@@ -0,0 +1,43 @@
+CREATE ()§
+CREATE( )§
+CYPHER 3.0 MATCH (n) RETURN n§
```

Comment:
I needed some non-Cypher character to use as a delimiter between the queries, and used this one.